### PR TITLE
Restrict Supported alg

### PR DESCRIPTION
**Pull request by OR13**
```
| kty | crvOrSize | alg    |
| --- | --------- | ------ |
| OKP | Ed25519   | EdDSA  |
| EC  | secp256k1 | ES256K |
| RSA | 2048      | PS256  |
| EC  | P-256     | ES256  |
```

Everything else will throw an error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,14 @@
...
 
 | kty | crvOrSize | alg    |
 | --- | --------- | ------ |
 | OKP | Ed25519   | EdDSA  |
 | EC  | secp256k1 | ES256K |
 | EC  | P-256     | ES256  |
-| RSA | \*        | PS256  |
-
-Other JWS are supported when `alg` is provided by the consuming library.
+| RSA | 4096      | PS256  |
```
Comment (msporny) [inline README.md]: Nope, overkill... 2048 for a 2020 suite... you won't need the next step up until around 2030 at which point we shouldn't be using RSA anyway.

File: README.md [hunk 2]
```diff
@@ -4,16 +4,14 @@
...
 ## Supported JWS Algs
 
-If an `alg` is not specified, this library will assume the following:
+The expected alg will be determined by the following table.
 
 | kty | crvOrSize | alg    |
 | --- | --------- | ------ |
 | OKP | Ed25519   | EdDSA  |
 | EC  | secp256k1 | ES256K |
 | EC  | P-256     | ES256  |
```
Comment (msporny) [inline README.md]: I won't make the "this is totally broken and has a backdoor" argument here... I expect others will make it, and unfortunately, US Govt. has mandated it in some scenarios.

Comment (OR13) [reply README.md]: I can't help what works in secure enclaves, Azure Key Vault of Non Extractable Web Crypto, or what is required to work with Bitcoin or Ethereum... I'm trying to strike a balance between "safe recommendations" and "has any support in the wild".... But I hear you. If you want to lock down further, you should probably use a more specific suite... I would consider recommending that at the top of this...

File: README.md (hunk 1 above)
Comment (OR13) [reply README.md]: Agreed.

File: README.md (hunk 2 above)
Comment (OR13) [reply README.md]: Added warning to readme: https://github.com/transmute-industries/lds-jws2020/pull/7/commits/11cf26efbabf15f310d38244397b4c816bcf5f9e

**Pull request merged**